enterprise/backend/src/metabase_enterprise/metabot_v3/tools/api.clj

```diff
@@ -718,18 +718,16 @@
 (defn- enforce-authentication
   "Middleware that returns a 401 response if no `ai-session` can be found for  `request`."
   [handler]
-  (with-meta
-   (fn [{:keys [headers] :as request} respond raise]
-     (if-let [{:keys [user metabot-id]} (-> headers
-                                            (get "x-metabase-session")
-                                            decode-ai-service-token)]
-       (request/with-current-user user
-         (handler (assoc request :metabot-v3/metabot-id metabot-id) respond raise))
-       (if (:metabase-user-id request)
-         ;; request relying on metabot-id are going to fail
-         (handler request respond raise)
-         (respond request/response-unauthentic))))
-   (meta handler)))
```

Comment:
Are we intentionally dropping the metadata on these update handlers? Were they simply there for no good reason?

Reply:
It was there for the old way we generated OpenAPI dox using the old version of `defendpoint` -- I reworked that stuff a while back (check out `handler-with-open-api-spec`) and this is no longer needed. I thought I removed all the places we were doing this but I missed the ones in `server` I guess